src/components/QuestionTypeSelector/QuestionTypeSelector.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import Form from 'react-bootstrap/Form'
+import ListGroup from 'react-bootstrap/ListGroup'
+import * as Icon from 'react-bootstrap-icons'
+import classNames from 'classnames'
+import './QuestionTypeSelector.scss'
+
+
```

Comment:
```suggestion
const Row = ({ icon, children }) => (
   <Item><icon/>&nbsp;{ children }</Item>
)
```
And replace all instances of `<ListGroup.Item>...</ListGroup.Item>` with this: `<Row icon={List}>Multiple Choice</Row>`

Reply:
I don't think this is worth the effort. All you are doing is moving the icon to be a attribute of Item rather than a child.